.gitlab-ci.yml

```diff
@@ -0,0 +1,107 @@
+
+stages:
+  - lint
+  - test
+  - build
+
+variables:                         &default-vars
+  GIT_STRATEGY:                    fetch
+  GIT_DEPTH:                       100
+  CARGO_INCREMENTAL:               0
+  CARGO_TARGET_DIR:                "/ci-cache/${CI_PROJECT_NAME}/targets/${CI_COMMIT_REF_NAME}/${CI_JOB_NAME}"
+  CI_IMAGE:                        "paritytech/ci-linux:production"
```

Comment:
what image is better here actually?
https://github.com/paritytech/scripts/blob/master/dockerfiles/ci-linux/Dockerfile or
https://github.com/paritytech/scripts/blob/master/dockerfiles/bridges-ci/Dockerfile
@tomusdrw maybe you can tell?
Or is it just a question of compatibility with Substrate?